tests/pr_test.go

```diff
@@ -258,3 +260,60 @@ func getCOSInstanceClient(apiKey, serviceInstanceID, authEndpoint, serviceEndpoi
 		WithS3ForcePathStyle(true)
 	return s3.New(sess, conf)
 }
+
+// A single function to test all DA solutions.
+func TestRunSolutions(t *testing.T) {
+	t.Parallel()
+
+	instanceOptions := setupOptions(t, "cos-da", solutionInstanceDir)
+	instanceOptions.TerraformVars = map[string]interface{}{
+		"existing_resource_group": true,
```

Comment:
in order to ensure we test resource group creation in the DA, please set this to false and use the options.Prefix value for `resource_group_name`